src/book.py

```diff
@@ -106,7 +106,7 @@ def clean(self, lemmatize=True, deromanize=False, lemma_pos='v', inplace=False):
         if inplace:
             self._text = cleaned
         else:
-            return BookText(filepath=None, rawtext=cleaned)
+            return BookText(filepath=None, rawtext=cleaned, author=self._author)
```

Comment:
```suggestion
            return BookText(filepath=None, rawtext=cleaned, author=self._author,
                            title=self._title, meta=self._meta)
```

Reply:
I have added meta and title data to line 109 as well as changed the author call from self._author to self.author . These changes are reflected in commit bbb2f13.